src/main/scala/nl.knaw.dans.easy.deposit/docs/AgreementsXml.scala

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (C) 2018 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.deposit.docs
+
+import org.joda.time.DateTime
+
+import scala.util.{ Failure, Success, Try }
+import scala.xml.Elem
+
+object AgreementsXml {
+  def apply(userId: String, dateSubmitted: DateTime, dm: DatasetMetadata): Try[Elem] = {
```

Comment:
👍 for this pattern

@alexal14 can you also do a similar style for `files.xml`?